### PR TITLE
Integrate with Travis CI and coverals for continuous integration and …

### PR DESCRIPTION
**Pull request by drum-IT**
…code coverage.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+[![Coverage Status](https://coveralls.io/repos/github/drum-IT/multiCLI/badge.svg?branch=master)](https://coveralls.io/github/drum-IT/multiCLI?branch=master)
```
Comment (tabuckner) [inline README.md:1]: do we want to move our default branch to develop to add an extra layer of security?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+sudo: false
+language: node_js
+node_js:
+  - "8"
+branches:
+  only:
+    - master
```
Comment (tabuckner) [inline .travis.yml:7]: i think the team might benefit from also seeing their travis build status for feature branches, what do you think?

**Review by tabuckner — APPROVED** [review]
approved with some questions 🚀

File: README.md (hunk 1 above)
Comment (drum-IT) [reply README.md:1]: Great idea. I'm going to merge this to wrap up initial integration, and then we can get started on that.

**Pull request merged**